src/Controller/RequestController.php

```diff
@@ -91,4 +93,29 @@ private function issetPost(): array
         }
         return $myPost;
     }
+
+    public function takeCare()
+    {
+        $errors = [];
+
+        if ($_SERVER['REQUEST_METHOD'] === 'POST' && isset($_POST['userId']) && isset($_POST['requestId'])) {
+            if ($_POST['userId'] != '-- --') {
+                $answererId = trim($_POST['userId']);
+                $requestId = trim($_POST['requestId']);
+                $answererId = filter_var($answererId, FILTER_VALIDATE_INT);
+                $requestId = filter_var($requestId, FILTER_VALIDATE_INT);
```

Comment:
filter_var returns false if the filter fails (in this case if it's not an int)
But you don't check the values it returns so in fact you don't check